src/pages/AddPersonalBankAccountPage.js

```diff
@@ -28,15 +28,15 @@ const AddPersonalBankAccountPage = props => (
         <KeyboardAvoidingView>
             <HeaderWithCloseButton
                 title={props.translate('bankAccount.addBankAccount')}
-                onCloseButtonPress={Navigation.dismissModal}
+                onCloseButtonPress={() => Navigation.goBack()}
```

Comment:
No need to create an anonymous func here. Just `onCloseButtonPress={Navigation.goBack}` is fine